plat/common/aarch64/plat_common.c

```diff
@@ -56,11 +56,13 @@ void bl32_plat_enable_mmu(uint32_t flags)
 
 void bl31_plat_runtime_setup(void)
 {
+#ifndef DEBUG
```

Comment:
I think DEBUG is always defined, to either 0 or 1. You might want to use `#if !DEBUG` to get the intended effect.